src/olympia/files/utils.py

```diff
@@ -151,6 +151,9 @@ class RDFExtractor(object):
     TYPES = {'2': amo.ADDON_EXTENSION, '4': amo.ADDON_THEME,
              '8': amo.ADDON_LPAPP, '64': amo.ADDON_DICT,
              EXPERIMENT_TYPE: amo.ADDON_EXTENSION}
+    # Langpacks and dictionaries, if the type is properly set, are always
+    # considered restartless.
+    ALWAYS_RESTARTLESS_TYPES = ('64', '8')
```

Comment:
Yeah I wanted to remove the numbers hardcoding, and was trying to figure out something clean, but nothing was really satisfying. I'll go with your suggestion, it's not *much* more readable, but I think it's still an improvement.